### PR TITLE
Specify DB user search_path

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 1]
```diff
@@ -598,6 +597,7 @@ The cluster database user is the user specified in `db.cluster.username` in the
 GRANT USAGE ON SCHEMA CONFIG to <CLUSTER-DB-USER>;
 GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA CONFIG to <CLUSTER-DB-USER>;
 GRANT USAGE, SELECT ON ALL SEQUENCES IN SCHEMA CONFIG TO <CLUSTER-DB-USER>;
+ALTER ROLE "user" SET search_path TO config;
```
Comment (kyriathar) [inline content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md:600]:
```suggestion
ALTER ROLE <CLUSTER-DB-USER> SET search_path TO CONFIG;
```

Comment (kyriathar) [reply content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md:600]: Should we introduce a placeholder for the config name for consistency?

File: content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 2]
```diff
@@ -608,9 +608,11 @@ Create the RBAC and Crypto users and grant access as follows:
 CREATE USER <RBAC-USERNAME> WITH ENCRYPTED PASSWORD '<RBAC-PASSWORD>';
 GRANT USAGE ON SCHEMA RBAC to <RBAC-USERNAME>;
 GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA RBAC to <RBAC-USERNAME>;
+ALTER ROLE "rbac_user" SET search_path TO rbac;
```
Comment (kyriathar) [inline content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md:611]:
```suggestion
ALTER ROLE <RBAC-USERNAME> SET search_path TO RBAC;
```

File: content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 3]
```diff
@@ -608,9 +608,11 @@ Create the RBAC and Crypto users and grant access as follows:
...
 CREATE USER <CRYPTO-USERNAME> WITH ENCRYPTED PASSWORD '<CRYPTO-PASSWORD>';
 GRANT USAGE ON SCHEMA CRYPTO to <CRYPTO-USERNAME>;
 GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA CRYPTO to <CRYPTO-USERNAME>;
+ALTER ROLE "crypto_user" SET search_path TO crypto;
```
Comment (kyriathar) [inline content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md:615]:
```suggestion
ALTER ROLE <CRYPTO-USERNAME> SET search_path TO CRYPTO;
```

File: content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md [hunk 4]
```diff
@@ -621,6 +623,7 @@ The state manager database users are configured by the username specified in the
 GRANT USAGE ON SCHEMA STATE_MANAGER TO <STATE_MANAGER_USERNAME>;
 GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA STATE_MANAGER TO <STATE_MANAGER_USERNAME>;
 GRANT USAGE, SELECT ON ALL SEQUENCES IN SCHEMA STATE_MANAGER TO <STATE_MANAGER_USERNAME>;
+ALTER ROLE "user" SET search_path TO state_manager;
```
Comment (kyriathar) [inline content/en/platform/corda/5.2/deploying-operating/deployment/deploying/manual-bootstrapping.md:626]:
```suggestion
ALTER ROLE <STATE_MANAGER_USERNAME> SET search_path TO STATE_MANAGER;
```

**Review by kyriathar — APPROVED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**